content/src/content/jcr_root/apps/core/wcm/components/accordion/v1/accordion/amp.html

```diff
@@ -0,0 +1,37 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<amp-accordion data-sly-use.accordion="com.adobe.cq.wcm.core.components.models.Accordion"
+     class="cmp-accordion"
+     data-cmp-is="accordion"
+     data-sly-attribute.expand-single-section="${accordion.singleExpansion}">
+      <section data-sly-test="${accordion.items.size > 0}"
+        data-sly-repeat.item="${accordion.items}"
+        class="cmp-accordion__item"
+        data-cmp-hook-accordion="item"
+        data-sly-attribute.expanded="${item.name in accordion.expandedItems}">
+        <h3 data-sly-element="${accordion.headingElement @ context='elementName'}"
+            class="cmp-accordion__header">
+            <span class="cmp-accordion__title">${item.title}</span>
+            <span class="cmp-accordion__icon"></span>
+        </h3>
+        <div data-sly-resource="${item.name @ decorationTagName='div'}"
+            data-cmp-hook-accordion="panel"
+            class="cmp-accordion__panel${item.name in accordion.expandedItems ? ' cmp-accordion__panel--expanded' : ' cmp-accordion__panel--hidden'}"
```

Comment:
The `cmp-accordion__panel--expanded` and `cmp-accordion__panel--hidden` classes can be removed here as they don't get toggled in the amp JS.